lib/deck.rb

```diff
@@ -0,0 +1,23 @@
+class Deck
+
+  attr_reader :cards, :count, :cards_in_category
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.count
```

Comment:
Instead of calling to `cards`, call to `@cards`. When you call to cards, it calls to the attr_Reader, which returns the instance variable. Since you can call directly to the instance variable, do that, and cut out the middle man.